server/handlebars/views/main.hbs

```diff
@@ -13,26 +12,25 @@
             <summary id="candidate-title"><span>Warning! Unapproved Report</span></summary>
             <div id="candidate-content-container">
                 The information in this report is generated from candidate tests developed and run by the <a href="http://aria-at.w3.org">ARIA-AT Project</a>.
-                ARIA-AT tests are in review by assistive technology developers and lack consensus regarding:
+                Candidate ARIA-AT tests are in review by assistive technology developers and lack consensus regarding:
                 <ol>
                     <li>applicability and validity of the tests, and</li>
                     <li>accuracy of test results.</li>
                 </ol>
             </div>
         </details>
         {{else}}
-        <div id="embed-report-status-container">
-            <div id="candidate-title" class="recommended"><h3>Recommended Report</h3></div>
+        <details id="embed-report-status-container">
+            <summary id="candidate-title" class="recommended"><span>Recommended Report</span></summary>
```

Comment:
I'd be curious about what the community's thoughts would on this title. Should the disclaimer also be the reverse of the `CANDIDATE` disclaimer, so `Approved Report`? 

This is also done on the application's Reports pages as well.

Reply:
It would be good to have them review it. Can we promote one of the reports to a recommended state on the sandbox?